sources/osg/BufferArray.js

```diff
@@ -102,6 +102,12 @@ BufferArray.flushAllDeletedGLBufferArrays = function(gl) {
     }
 };
 
+BufferArray.lostContext = function(gl) {
```

Comment:
Naming here is not clear. lostContext doesn't sound like an action, if it's something called as a callback, maybe call it onContextLost, else, maybe call it, clearContext, discardContext or something like that.

Reply:
ok for onLostContext then ?
("LostContext" is name of the event in webgl)